-------------------------------------------------------------------------------
Test set: com.studentapp.junit.testTwo.SecondSerenityTest
-------------------------------------------------------------------------------
Tests run: 7, Failures: 1, Errors: 2, Skipped: 3, Time elapsed: 3.974 sec <<< FAILURE! - in com.studentapp.junit.testTwo.SecondSerenityTest
thisIsATestWithFileDoesNotExist(com.studentapp.junit.testTwo.SecondSerenityTest)  Time elapsed: 0.335 sec  <<< ERROR!
java.io.FileNotFoundException: E:\file.txt (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileReader.<init>(FileReader.java:72)
	at com.studentapp.junit.testTwo.SecondSerenityTest.thisIsATestWithFileDoesNotExist(SecondSerenityTest.java:65)

thisIsATestWithError(com.studentapp.junit.testTwo.SecondSerenityTest)  Time elapsed: 0.346 sec  <<< ERROR!
java.lang.ArithmeticException: / by zero
	at com.studentapp.junit.testTwo.SecondSerenityTest.thisIsATestWithError(SecondSerenityTest.java:59)

thisIsAFailing(com.studentapp.junit.testTwo.SecondSerenityTest)  Time elapsed: 1.09 sec  <<< FAILURE!
java.lang.AssertionError: 1 expectation failed.
Expected status code <500> but was <200>.

	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:249)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)
	at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:656)
	at sun.reflect.GeneratedMethodAccessor175.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:190)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:58)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:168)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:125)
	at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
	at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:133)
	at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)
	at com.studentapp.junit.testTwo.SecondSerenityTest.thisIsAFailing(SecondSerenityTest.java:44)

